crypto/mem.c

```diff
@@ -38,7 +38,8 @@ static TSAN_QUALIFIER int free_count;
 #  define LOAD(x)      tsan_load(&x)
 # endif /* TSAN_REQUIRES_LOCKING */
 
-static char *md_failstring;
+static char md_failbuf[CRYPTO_MEM_CHECK_MAX_FS + 1];
+static char *md_failstring = md_failbuf;
```

Comment:
(minor) Why not initialise this to `NULL` (or leave it uninitialised, like it was before)? Then we keep the behaviour that if this is `NULL` there is no `OPENSSL_MALLOC_FAILURES`

Reply:
ack, will do